master/networking/mtu.md

```diff
@@ -3,127 +3,138 @@ title: Configuring MTU
+| 1450 (OpenStack VXLAN) | 1450                  | 1430                                       | 1400                                    |
 
-Passing in the environment variable `FELIX_IPINIPMTU` (or `FELIX_VXLANMTU`) when running the
-`{{site.nodecontainer}}` container will set the MTU for Felix to use.
+##### Recommended MTU for overlay networking
 
-When using the Kubernetes self-hosted manifests, Felix derives the IP-in-IP value from
-the `veth_mtu` field of the calico-config ConfigMap, which is set to `1440` by default.
+The extra overlay header used in IP in IP and VXLAN protocols, reduces the minimum MTU by the size of the header. (IP in IP uses a 20-byte header, and VXLAN uses a 50-byte header). Therefore, we recommend the following:
 
-### Setting tunnel MTU with calicoctl
+- If you use VXLAN anywhere in your pod network, configure MTU size as “physical network MTU size minus 50”. 
+- If you use only IP in IP, configure MTU size as “physical network MTU size minus 20”
+- Set the workload endpoint MTU and the tunnel MTUs to the same value (so all paths have the same MTU)
 
-To set the IP-in-IP MTU value for all {{site.prodname}} nodes in your cluster, use the
-following command to retrieve the current Felix settings.
+##### MTU for flannel networking
 
-```bash
-calicoctl get felixconfig default --export -o yaml > felix.yaml
-```
+When using flannel for networking, the MTU for network interfaces should match the MTU of the flannel interface. If using flannel with VXLAN, use the “Calico MTU with VXLAN” column in the table above for common sizes. 
+
+#### Configure MTU for workloads
+  
+When you set the MTU it applies to new workloads. To apply MTU changes to existing workloads, you must restart calico nodes. Restarting the calico/node pods also updates any Calico tunnel network interfaces on that node. 
 
-Modify ipipMTU (or vxlanMTU) to the intended integer value.
+Edit `calico-config ConfigMap` with the veth MTU value for your environment. For example: 
 
-```bash
-vim felix.yaml
 ```
+# The CNI network configuration to install on each node.
+  cni_network_config: |-
+    {
+        "name": "k8s-pod-network",
+        "cniVersion": "0.1.0",
+        "type": "calico",
+        "etcd_endpoints": "__ETCD_ENDPOINTS__",
+        "etcd_key_file": "__ETCD_KEY_FILE__",
+        "etcd_cert_file": "__ETCD_CERT_FILE__",
+        "etcd_ca_cert_file": "__ETCD_CA_CERT_FILE__",
+        "log_level": "info",
+        "mtu": 1440,
+```
+
+#### Configure MTU for overlay networking
+
+If you are using IP in IP and/or VXLAN for Calico overlay networking, set the tunnel MTU to match the value that you configured for the veth MTU. 
 
-Replace the current felixconfig settings.
+Edit `calico-config ConfigMap` with the MTU tunnel values for your environment. For example: 
 
-```bash
-calicoctl replace -f felix.yaml
 ```
+# Set MTU for tunnel device used if ipip is enabled
```

Comment:
Weird mismatched indentation here

Reply:
Not seeing this paragraph in my version of this doc.

Reply:
This is not showing in my version.